Cabal/src/Distribution/Simple/Glob.hs

```diff
@@ -14,59 +14,247 @@
 -- Portability :  portable
 --
 -- Simple file globbing.
-module Distribution.Simple.Glob
-  ( GlobSyntaxError (..)
-  , GlobResult (..)
-  , matchDirFileGlob
-  , matchDirFileGlobWithDie
-  , runDirFileGlob
-  , fileGlobMatches
-  , parseFileGlob
-  , explainGlobSyntaxError
-  , isRecursiveInRoot
-  , Glob
-  ) where
+module Distribution.Simple.Glob where
```

Comment:
Can we keep the export list? We probably want to avoid exporting implementation details if possible

Reply:
Done.